apps/api/src/app/bridge/usecases/sync/sync.usecase.ts

```diff
@@ -152,6 +152,7 @@ export class Sync {
               active: this.castToAnyNotSupportedParam(workflow.options)?.active ?? true,
               critical: this.castToAnyNotSupportedParam(workflow.options)?.critical ?? false,
               preferenceSettings: this.castToAnyNotSupportedParam(workflow.options)?.preferenceSettings,
+              preference: workflow.preference,
```

Comment:
We should use the plural `preferences` across the board (aligns with existing `tags`, plus it's preferable to keep domain language pluralized, as we do for REST API resource naming)